client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -211,16 +230,30 @@ const getAddressErrors = ( addressData, countriesData ) => {
 		}
 	} );
 
-	if ( countriesData[ country ] ) {
-		if (
-			includes( ACCEPTED_USPS_ORIGIN_COUNTRY_CODES, country ) &&
-			! /^\d{5}(?:-\d{4})?$/.test( postcode )
-		) {
-			errors.postcode = translate( 'Invalid ZIP code format' );
-		}
+	if (
+		includes( ACCEPTED_USPS_ORIGIN_COUNTRIES, country ) &&
+		! /^\d{5}(?:-\d{4})?$/.test( postcode )
+	) {
+		errors.postcode = translate( 'Invalid ZIP code format' );
+	}
+
+	if ( ! state && hasStates( appState, country, siteId ) ) {
+		errors.state = translate( 'This field is required' );
+	}
 
-		if ( ! isEmpty( countriesData[ country ].states ) && ! state ) {
-			errors.state = translate( 'This field is required' );
+	if ( shouldValidatePhone ) {
+		// EasyPost requires an origin phone number for international shipments.
+		// This validation ensures that EasyPost will accept it, even though the phone may be invalid.
+		if ( ! phone ) {
+			errors.phone = translate( 'Origin phone number is required for international shipments' );
+		} else if (
+			10 !==
+			phone
+				.split( /\D+/g )
+				.join( '' )
+				.replace( /^1/, '' ).length
+		) {
+			errors.phone = translate( 'Invalid phone number' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Invalid Phone Number' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-intl-labels-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).